docs/tutorials/external-events/associating-derivation-groups.mdx

```diff
@@ -0,0 +1,136 @@
+
+By default, a dedicated 'External Events Row' is **not** created *unless* the user loads a plan that already existed and has at least one derivation group previously linked. In our case, because this is a new plan we must create our own row to see the events. Alternatively, a filter for the event types can also be added to a pre-existing row.
+
+First, switch to the `Activity, Resource, & Event Types` tab.
+
+<figure>
+  <img alt="Switch to the 'Activity, Resource, & Event Types' tab" src={externalEventsTutorialAddingRowStep1} />
+  <figcaption><i>Switch to the 'Activity, Resource, & Event Types'</i></figcaption>
+</figure>
+
+Click on the `Events` tab in the pane. The external event types from the example events should appear inside of the pane - these can be dragged to pre-existing rows to add as a filter, or by using the `+` icon when hovering over the type entry.
+
+For this tutorial, we'll create a new row entirely. We can also expedite the process by using the `Add Filter to Row` button at the top of the pane, and then clicking the `New Row +` button at the bottom of the list. This will create a new row, with all available external event types filtered *in*.
+
+<figure>
+  <img alt="Click 'Add Filter to Row' and then 'New Row +'" src={externalEventsTutorialAddingRowStep2} />
+  <figcaption><i>Click 'Add Filer to Row' and then 'New Row +'</i></figcaption>
+</figure>
+
+We can now see our example events on the timeline! There are 4 **derived** events shown, these represent the set of events that are *derived* from all the sources that existent within the derivation group.
+
+<figure>
+  <img alt="External Events Row" src={externalEventsTutorialAddingRowStep3} />
+  <figcaption><i>External Events Row</i></figcaption>
+</figure>
+
+In the next section, we'll look at the options available when editing a layer on the timeline.
+
+## Editing the External Event timeline layer
+Clicking the three dots on the bottom right of the layer and then clicking **Edit Row** will bring up the layer timeline editor:
+
+<figure>
+  <img alt="Opening the layer row settings" src={externalEventsTutorialOpeningRowSettings} />
+  <figcaption><i>Click the three dots</i></figcaption>
+</figure>
+
+<figure>
+  <img alt="Opening the layer 'Edit Row' option" src={externalEventsTutorialOpeningEditRow} />
+  <figcaption><i>Click 'Edit Row'</i></figcaption>
+</figure>
+
+<figure>
+  <img alt="The Timeline Editor panel" src={externalEventsTutorialTimelineEditor} />
+  <figcaption><i>The Timeline Editor for the row will appear!</i></figcaption>
+</figure>
+
+External events appear in the same layer type as activities, and as such share options with them inside of the timeline row editor.
+
+There is a single option that is specific to external events: `Group By`. This allows the user to display the external events either as being grouped underneath the their respective external sources, or grouped underneath their respective external event types.
```

Comment:
For some reason when an external events row is added via the method in this tutorial, the "Group By" option does not appear in the Timeline Editor as expected. I think this is a bug, because it *does* appear in the automatically generated External Events row that shows up if you refresh the page. It's not a showstopper bug, but I'm going to comment out the paragraphs that mention "group by" here to avoid confusion & @JosephVolosin we can look into this for a follow up fix.